src/Microsoft.DotNet.XHarness.iOS.Shared/TestImporter/Templates/Managed/Resources/src/common/ApplicationOptions.cs

```diff
@@ -70,6 +73,7 @@ public ApplicationOptions()
                 { "autostart", "If the app should automatically start running the tests.", v => AutoStart = true },
                 { "hostname=", "Comma-separated list of host names or IP address to (try to) connect to", v => HostName = v },
                 { "hostport=", "HTTP/TCP port to connect to.", v => HostPort = int.Parse (v) },
+                { "use_tcp_tunnel", "Use a tcp tunnel to connect to the host.", v => UseTcpTunnel = true },
```

Comment:
This reminds me - Marek wanted to get rid of Mono.Options from the app bundle. Should we have an issue for that or it's not happening anymore?

Reply:
We should create an issue, means we will need to parse things manually from the command line that mlaunch sends: https://github.com/dotnet/xharness/issues/72